tests/agent/handlers/nodesHolePunchRequest.test.ts

```diff
@@ -0,0 +1,147 @@
+import type * as quicEvents from '@matrixai/quic/dist/events';
+import type { Host as QUICHost } from '@matrixai/quic';
```

Comment:
This type is no longer necessary. We changed js-quic to take `string` for `host` and `number` for `port`.

This avoids needing all these type conversions.

Reply:
I missed them here, removed.